MAUI/DateTimePicker/formatting.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: Formatting in .NET MAUI Date Time Picker control | Syncfusion
```

Comment:
title: Formatting in .NET MAUI Date Time Picker Control | Syncfusion

Reply:
Content modified.